# Gene Sequencing Techniques
There are various techniques that can be employed in order to sequence DNA. 


# The Maxam Gilbert Technique
The first method, known as the Maxam Gilbert method, was developed in 1977.
It was not an effective way of sequencing DNA, however.
In MGS, DNA extraction was, inevitably, the first step. The DNA is then denatured to give single stranded DNA. The phosphate (5P') at the end of the DNA is removed and labelled. An enzyme called phosphatase removes the phosphate from the DNA, and simultaneously, the kinase adds the 32P to the 5' end of it.

4 different chemicals are used to cleave DNA at four different positions; hydrazine and hydrazine NaCl are selectively attack pyrimidine nucleotides while dimethyl sulfate and piperidine attack purine nucleotides.

Hydrazine: T + C\
Hydrazine NaCl: C\
Dimethyl sulfate: A + G\
Piperidine: G

An equal volume of 4 different ssDNA samples is taken into 4 different tubes each containing 4 different chemicals. The samples are incubated for some time and electrophoresed in polyacrylamide gel electrophoresis. The scalability of MGS is poor, and that makes it harder to use.

# Sanger's Method
This was the first ever method proposed for sequencing genes. Developed by and named after Nobel Prize winning scientist Frederick Sanger, this scientific accomplishment changed the face of biotech.

The DNA to be sequenced is prepared in a single strand. In the presence of the 4 deoxyribonucleic triphosphates A, G, C and T, the polymerase extends the primer by adding the complementary dNTP to the strand of this template DNA. To determine which nucleotide is incorporated into this chain of nucleotides, a simple test is performed. Four dioxynucleotide phosphates, namely ddATP, ddCTP and ddTTP and ddGTP. They are labelled with fluorescent dye and are used to terminate the synthesis reaction.

A ddNTP has an oxygen removed from the ribonucleotide and therefore cannot form a bond with the next nucleotide, unlike dNTPs.

All in all, this sequencing technique has 6 steps

# Solexa Sequencing
This technique uses a concept called synthesis. Sequencing templates are immobilized on a proprietary flow cell surface. Unlabeled nucleotides and enzymes are added to initiate solid-phase bridge amplification. The enzyme incorporates nucleotides to build double-stranded bridges on the solid-phase substrate. Denaturation leaves single-stranded templates anchored to the substrate. Several million dense clusters of double-stranded DNA are generated in each channel of the flow cell. The first sequencing cycle begins by adding four labeled reversible terminators, primers, and DNA polymerase. After laser excitation, the emitted fluorescence from each cluster is captured, and the first base is identified. The next cycle repeats the incorporation of four labeled reversible terminators, primers, and DNA polymerase. After laser excitation, the image is captured as before, and the identity of the second base is recorded. The sequencing cycles are repeated to determine the sequence of bases in a fragment, one base at a time. The data are aligned and compared to a reference, and sequencing differences are identified.

# References:

1. Gauthier, Michel G. Simulation of polymer translocation through small channels: A molecular dynamics study and a new Monte Carlo approach. Diss. University of Ottawa (Canada), 2008.
2. Sikkema‐Raddatz, Birgit, et al. Targeted next‐generation sequencing can replace Sanger sequencing in clinical diagnostics. Human mutation 34.7 (2013): 1035-1042.
3. https://explorebiotech.com/genome-sequencing-methods-and-applications/ by Bishal Khatiwada in Genetics.
